etc/profile-m-z/parsecd.profile

```diff
@@ -0,0 +1,38 @@
+
+# Firejail profile for Parsec
+# Description: Remote desktop application focused on gaming and other 3D applications
+# This file is overwritten after every install/update
+
+# Persistent local customizations
+include parsecd.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.parsec
+whitelist ${HOME}/.parsec
+whitelist /usr/share/parsec
+include whitelist-common.inc
+include whitelist-usr-share-common.inc
+
+# Due to the nature of parsec, the following directives will not work:
+# - no3d
+# - novideo
+# - noaudio
+# - noinput (it does remote passthrough stuff for gamepads)
+# - private-dev (because of the above)
+apparmor
```

Comment:
You said it needs executable `$HOME`, but here you set `apparmor`?

Reply:
include disable-exec.inc will break it, but apparmor will not. I have no idea why. The  thing in question in ~/.parsec: `parsecd-150-86e.so: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, BuildID[sha1]=1a9684ced0e2b6198904b4fc070d8ba09db27fcf, not stripped`

Reply:
Do you use AA? Is it enabled in your kernel? And the firejail profile is loaded?

Reply:
Yes I do.

```
apparmor module is loaded.
63 profiles are loaded.
63 profiles are in enforce mode.
   /usr/local/bin/parsecd (1070703) firejail-default//&unconfined   /usr/local/bin/parsecd (1070703) firejail-default//&unconfined
```

Reply:
Interesting

Reply:
I've seen and used this AA 'trickery' before in combination with Firejail. BUT I have enabled stuff in /etc/apparmor.d/local/firejail-default for doing so:

`
owner @{HOME}/some/path/** ix,
`

Perhaps @NetSysFire uses a similar setup?

Reply:
No. I never touched firejail-default.